finalPrep/modZeroHero.js

```diff
@@ -41,13 +83,31 @@
 //   - Anything dangerLevel that is between 10 and 50 should result in printing the saveTheDay string to the console
 //   - If the dangerLevel is below 10, it means it is not worth your time and should result in printing the string "Meh. Hard pass." to the console.
 
+```
+var dangerLevel = 0;
+var saveTheDay = "Never Fear. I am here to save the day!";
+var badExcuse = "I am so sorry. My 🚨 alarm didn't go off.";
+
```

Comment:
Nice job here. Just a tip- sometimes, I get mixed up when I name variables the same exaxt things as the params a function takes. Any variables can be passed into the params of `assessSituation` when we call that function.